packages/apps/public/locales/en/apps-config.json

```diff
@@ -50,6 +50,7 @@
   "rpc.nodle-arcadia": "Arcadia",
   "rpc.nodle-main": "Nodle",
   "rpc.phala": "Phala PoC-3",
+  "rpc.phoenix": "PHOENIX Mashnet",
```

Comment:
This doesn't quite seem to match up above? All lowercase there, all uppercase here? (Small nitpick, but the i18n generation will just warn, on this one)

Reply:
Thank you. I have fixed the issues and submitted the changes.